src/devices/SenseHat/SenseHatTextRotation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.SenseHat
+{
+    /// <summary>
+    /// Text rotation when rendering text, counterclockwise.
```

Comment:
on the counterclockwise : you have degrees in the enum, so not sure how this comments align

Reply:
Not sure if I understand... The enum is named `Rotate_90_degrees`, `Rotate_180_degrees`, etc., so in the mathematical sense indicates this to be a positive (counterclockwise) rotation, but is it not helpful to point this out in the comment? Could you give an example of what kind of comment you had in mind?

Reply:
By convention, the rotations are clockwise so if it's 90 degrees, it's 90 degrees clockwise. 270 degrees would be 90 degrees counterclockwise then.

Reply:
Sorry, still confused. The angle of rotation currently currently follows the mathematical convention (counterclockwise, i.e., https://en.wikipedia.org/wiki/Rotation_matrix; see "Direction"). Could you point to the convention that should be followed for the IoT library?

At the moment you get this:

![image](https://github.com/dotnet/iot/assets/47704763/835c13c4-6a01-4672-8427-5d4b3cc04b9f)

Reply:
my bad. You're right. All good then!